libr/core/cmd_print.c

```diff
@@ -5417,7 +5418,7 @@ l = use_blocksize;
 						core->num->value = r_core_print_disasm (core->print,
 							core, addr, block1, addrbytes * l, l, 0, 1, formatted_json, NULL, NULL);
 					} else {
-						eprintf ("Cannot allocate %d byte(s)\n", addrbytes * l);
+						eprintf ("Cannot allocate %lld byte(s)\n", addrbytes * l);
```

Comment:
```suggestion
						eprintf ("Cannot allocate %" PFMT64d " byte(s)\n", addrbytes * l);
```

Reply:
@ret2libc Can you push those modifications. I am not on my personnal computer.